lib/l10n/app_en.arb

```diff
@@ -331,6 +331,7 @@
   "@overview": {},
   "password": "Password",
   "@password": {},
+  "pending": "Pending",
```

Comment:
I would add in some metadata description for this to explain what or where this is used! It would be something like this:

```
"pending": "Pending",
"@pending": {
  "description": "Describes the subscription status for 'Pending'"
}
```

Reply:
All set!